packages/pushnotification/src/PushNotification.ts

```diff
@@ -18,6 +18,12 @@ const REMOTE_NOTIFICATION_RECEIVED = 'remoteNotificationReceived';
 const REMOTE_TOKEN_RECEIVED = 'remoteTokenReceived';
 const REMOTE_NOTIFICATION_OPENED = 'remoteNotificationOpened';
 
+/**
+ * @deprecated This package (@aws-sdk/pushnotifications) is on a deprecation path. Please see the following link for
```

Comment:
```suggestion
 * @deprecated This package (@aws-amplify/pushnotification) is on a deprecation path. Please see the following link for
```

Here and elsewhere in PR

Reply:
Phew good catch